docs/paper/dev/getting-started/plugin-yml.mdx

```diff
@@ -90,7 +90,7 @@ This will be shown in the plugin info commands.
 
 ### api-version
 
-The version of the Paper API that your plugin is using. This doesn't include the minor version.
+The version of the Paper API that your plugin is using. This doesn't include the minor version until 1.20.5. From 1.20.5 and onward, a minor version is supported.
```

Comment:
Looking at it again. Are we really sure to not follow semver here on purpose? The current docs say that the api version has no minor version, although it should be the patch.

I would suggest to actually phrase it like this, which applies more to the correct naming scheme.

```suggestion
The version of the Paper API that your plugin is using. This doesn't include the patch version (e.g. 1.13) until 1.20.5. From 1.20.5 and onward, a patch version (e.g 1.20.5) is supported.
```

Reply:
I brought this up on the Discord server (#docs-website) yesterday and was told that the community generally ignores the 1 in 1.xx.x, so we should probably stick to calling it `ignored.major.minor` as Minecraft isn't semantically versioned and it's what people expect